README.md

```diff
@@ -5,8 +5,19 @@ Self learning project
 ## Installation
 Clone repository
 
+## How to launch
+
 ```bash
 docker-compose up
+docker exec -it tm_php composer install
```

Comment:
Nice to have this composer instruction. IMHO good tip is run composer locally then composer get packages from cache (of course if you use they before in this or other project).